taxi/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from .models import Car, Manufacturer
+from crispy_forms.helper import FormHelper
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"
+    helper = FormHelper()

```

Comment:
The `helper` attribute should ideally be placed inside the `__init__` method of the form class to ensure it is initialized properly for each form instance. Consider moving `helper = FormHelper()` into an `__init__` method.